src/AuthorizationClient.js

```diff
@@ -240,6 +242,47 @@ class AuthorizationClient {
     return `${token}.${Utils.B64(multiSig)}`;
   }
 
+  async GenerateSignedLinkToken({containerId, versionHash, link}) {
+    ValidateObject(containerId);
+    const canEdit = await this.client.CallContractMethod({
+      contractAddress: Utils.HashToAddress(containerId),
+      methodName: "canEdit"
+    });
+
+    const { objectId } = Utils.DecodeVersionHash(versionHash);
+
+    if(!canEdit) {
+      throw Error(`Current user does not have permission to edit content object ${objectId}`);
+    }
+
+    const signerAddress = (this.client.signer && this.client.signer.address) || "";
```

Comment:
use this.client.CurrentAccountAddress()

https://eluv-io.github.io/elv-client-js/ElvClient.html#CurrentAccountAddress